.circleci/config.yml

```diff
@@ -47,7 +47,7 @@ jobs:
       - save_cache:
           key: v1-native-dependencies-{{ .Branch }}-{{ checksum "tablecloth-native.opam" }}
           paths:
-            - ~/opam
+            - ~/.opam
```

Comment:
I think you'll need to bump the version to v2 aswell - presumably the fallback cache already exists and we need to break it.